database/seeds/OrganizationRoleUiPermissionSeeder.php

```diff
@@ -0,0 +1,160 @@
+            "All Independent Activities" => "None",
+            "Export/Import Activities" => "None",
+            "All Projects" => "None",
+            "Import/Export Projects" => "None",
+            "Activity Types" => "None",
+            "Activity Items" => "None",
+            "Manage Users" => "None",
+            "Manage Roles" => "None",
+            "LMS Settings" => "None",
+            "LTI Tools" => "None",
+            "BrightCove" => "None",
+            "Project" => "Edit",
+            "Playlist" => "Edit",
+            "Activity" => "Edit",
+            "Team" => "Edit",
+            "Independent Activity" => "Edit",
+            "My Interactive Video" => "None"
+        ];
+
+        $roleTypes = OrganizationRoleType::whereIn('name', ['admin', 'course_creator', 'member', 'self_registered'])->get();
+        $this->uiModules = DB::table('ui_modules')->whereNotNull('parent_id')->pluck('id', 'title');
+
+        $uiModulePermissionsList = [];
+        $uiModulePermissionsResult = DB::table('ui_module_permissions')->get();
+        
+        foreach ($uiModulePermissionsResult as $uiModulePermission) {
+            $uiModulePermissionsList[$uiModulePermission->title][$uiModulePermission->ui_module_id] = $uiModulePermission->id;
+        }
+
+        $this->uiModulePermissions = $uiModulePermissionsList;
+
+        $this->domain = request()->getHttpHost();
+
+        foreach ($roleTypes as $roleType) {
+            if ($roleType->name === 'admin') {
+                $this->assignPermissions($adminUiPermissions, $roleType);
+            } else if ($roleType->name === 'course_creator') {
+                $this->assignPermissions($courseCreatorUiPermissions, $roleType);
+            } else if ($roleType->name === 'member') {
+                $this->assignPermissions($memberUiPermissions, $roleType);
+            } else if ($roleType->name === 'self_registered') {
+                $this->assignPermissions($selfRegisteredUiPermissions, $roleType);
+            }
+        }
+    }
+
+    function assignPermissions($uiPermissions, $roleType)
+    { 
+        $uiModulePermissionIds = [];
+       
+        foreach ($uiPermissions as $permissionName => $permissionType) {
+            
+            if ($this->domain === 'currikistudio.org') {
+                if ($permissionName === 'Team' && ($roleType->name === 'course_creator' || $roleType->name === 'self_registered')) {
+                    $permissionType = 'View';
+                }
+            } else if ($this->domain === 'oci.currikistudio.org') {
+                if ($permissionName === 'Independent Activity') {
+                    $permissionType = 'None';
+                } else if ($permissionName === 'My Interactive Video') {
+                    $permissionType = 'View';
+                }
+            }
+
+            if (isset($this->uiModules[$permissionName]) && $this->uiModulePermissions[$permissionType][$this->uiModules[$permissionName]]) {
```

Comment:
Move "$this->uiModules[$permissionName]" to variable.
Add "isset()" for $this->uiModulePermissions[$permissionType][$this->uiModules[$permissionName]]